femtools/Embed_Python_Fortran.F90

```diff
@@ -370,6 +422,50 @@ subroutine set_detectors_from_python_sp(function, function_len, dim, &
     result_z = lresult_z
 
   end subroutine set_detectors_from_python_sp
+
+  subroutine set_particles_from_python_sp(function, function_len, dim, &
```

Comment:
Add comment to this routine. How does it differ to set_particles_from_python? What is the _sp?